Meadowlark.net/Meadowlark.Net.Frontend.MinimalAPI/CrudHandler.cs

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Meadowlark.Net.Core;
+using static Meadowlark.Net.Core.FrontendFacade;
+
+namespace Meadowlark.Net.Frontend.MinimalAPI
+{
+  public static class CrudHandler
```

Comment:
We might want to call this a Controller, to help new people who are searching around for the Controllers.

This particular controller will work with two different API definitions: Descriptors and Resources. For now, "Crud" seems as good a unifying name as anything 😆 .